src/ui/component/dateRangePicker/dateRangePicker.tsx

```diff
@@ -0,0 +1,56 @@
+import { type FC, useCallback } from 'react'
+import { useTranslation } from 'react-i18next'
+import type { DayPickerProps } from '@/ui/component/dayPicker/dayPicker'
+import { DatePicker } from '@/ui/component/datePicker/datePicker'
+import './dateRangePicker.scss'
+
+export type DateRange = {
+  from: Date | undefined
+  to: Date | undefined
+}
+
+type DateRangePickerProps = Omit<DayPickerProps, 'title' | 'onSelect' | 'selected'> & {
+  onSelect: (dateRange: DateRange) => void
+  selected: DateRange
+}
```

Comment:
You couple this component with the `DayPicker` that means that it depends now on a specific contract. 
What about if we want to switch the current `DayPicker` component by another or to export it in a dedicated library ?

Reply:
yes if we want to switch daypicker component, we should avoid to couple them.
It's probably unlikely to happen. But if it does, we can simply rewrite the Datepicker props type, without changing the component API.

The goal here was to express the relationship between both component.

If you think that we shouldn't, we can definitely update it. Please let me know! :)